src/stores/PersonStore.ts

```diff
@@ -13,6 +17,11 @@ import axiosApiInstance from '@/services/ApiService';
 
 const personenApi: PersonenApiInterface = PersonenApiFactory(undefined, '', axiosApiInstance);
 const personenFrontendApi: PersonenFrontendApiInterface = PersonenFrontendApiFactory(undefined, '', axiosApiInstance);
+const personenKontexteApi: DbiamPersonenkontexteApiInterface = DbiamPersonenkontexteApiFactory(
```

Comment:
Ich habe in meinem 443 schon die Stores für Kontexte und Personen separiert, aber vielleicht lassen wir das erstmal so dann bauen wir das später um und wir haben dann auch weniger Konflikte.